common/test/acceptance/pages/lms/teams.py

```diff
@@ -102,3 +107,89 @@ def header_topic_description(self):
     def team_cards(self):
         """Get all the team cards on the page."""
         return self.q(css='.team-card')
+
+    def click_create_team_link(self):
+        """ Click on create team link."""
+        query = self.q(css=CREATE_TEAM_LINK_CSS)
+        if query.present:
+            query.first.click()
+            self.wait_for_ajax()
+
+    def click_search_team_link(self):
+        """ Click on create team link."""
+        query = self.q(css='.search-team-descriptions')
+        if query.present:
+            query.first.click()
+            self.wait_for_ajax()
+
+    def click_browse_all_teams_link(self):
+        """ Click on browse team link."""
+        query = self.q(css='.browse-teams')
+        if query.present:
+            query.first.click()
+            self.wait_for_ajax()
+
+    def wait_for_action_links_tobe_present(self):
+        """
+        Verifies that action link are visible.
+        """
+        return EmptyPromise(
```

Comment:
Nit: try using [`PageObject.wait_for`](https://bok-choy.readthedocs.org/en/latest/api_reference.html#bok_choy.page_object.PageObject.wait_for) rather than `EmptyPromise`.